restconnector/detail/RestZeqTranslator.cpp

```diff
@@ -23,54 +25,67 @@ RestZeqTranslator::~RestZeqTranslator()
 {
 }
 
-::zeq::Event RestZeqTranslator::translate( const std::string& body ) const
+zeq::Event RestZeqTranslator::translate ( const std::string& request, const std::string& body )
```

Comment:
Because of "vocabularySubscribed_.find( command );" Apparently you cannot do a find() on a const map... : (